app/views/groups/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="d-height d-flex flex-column justify-content-between" >
+
+<%= render partial: "shared/navbar", locals: { title: "CATEGORIES" } %>
+<br/>
+<%if user_signed_in?%>
+
+  <% if @groups.blank?%>
+    <p class="text-center">You don't have any category yet! Click below to add one.</p>
+  <% else %>
+<div id="groups" class="mt-5 ">
+<div class="mx-3 d-flex justify-content-end ">
+<span>$<%=@total_spending%></span>
+</div>
+  <% @groups.each do |group| %>
+    <%= render group %>
+  <% end %>
+</div>
+
+<%end%>
+<section class= "end">
```

Comment:
- [ ] The text is overflowing from the page. Please make sure to make the image responsive so that they take a fix width and don't break the UI.

I have attached a screenshot below of the home page. The category `text` and its `expense` is overflowing. Please fix that.

![Screenshot from 2022-02-20 00-37-45](https://user-images.githubusercontent.com/22434660/154817631-69f005cc-5d0e-4a6d-bea5-867ab5a503cd.png)